rate_counter/rate/models.py

```diff
@@ -1,5 +1,40 @@
 from __future__ import unicode_literals
+from django.contrib.auth.models import User
 
 from django.db import models
 
-# Create your models here.
+
+class Subject(models.Model):
+    """
+    Subject model representing subject from University schedule
+    """
+    name = models.CharField(max_length=100)
+    group = models.ForeignKey('StudentGroup', related_name='subjects')
+
+
+class Institute(models.Model):
+    """
+    Institute model representing Institute parsed from University
+    """
+    name = models.CharField(max_length=4)
+
```

Comment:
@Scandie Can't you do it like this:
1. get list of all instututes with identifier and name from NULP site (parse them)
2. Store it into DB(create objects of Institutes)
3. Iterate over Institutes objects , get their id or what, and use them in script?